setup.cfg

```diff
@@ -41,7 +41,9 @@ classifiers =
 [options]
 zip_safe = False
 python_requires = >=3.5
-install_requires = setuptools
+install_requires = 
+    setuptools
+    ninja
```

Comment:
We shouldn't require ninja for all scons installs. Please remove.

Reply:
The tool itself should check for availability like all such tools in it's exists().

Reply:
removed and added check in exists: 
c10303d
2419d9c